tools/extras/check_dependencies.sh

```diff
@@ -136,7 +138,7 @@ case $(uname -m) in
     # installed in an alternative location (this is unlikely).
     MKL_ROOT="${MKL_ROOT:-/opt/intel/mkl}"
     if [ ! -f "${MKL_ROOT}/include/mkl.h" ] &&
-         ! echo '#include <mkl.h>' | $CXX -I /opt/intel/mkl/include -E - >&/dev/null; then
+         ! echo '#include <mkl.h>' | $CXX -I /opt/intel/mkl/include -E - &>/dev/null; then
```

Comment:
here are two formats for redirecting standard output and standard error:

          &>word

   and

          >&word

   Of the two forms, the first is preferred.  This is semantically equivalent to

          >word 2>&1

Reply:
to make shellcheck happy